scripts/build_nim.sh

```diff
@@ -71,6 +72,7 @@ nim_needs_rebuilding() {
 		# support old Git versions, like the one from Ubuntu-18.04
 		git restore . 2>/dev/null || git reset --hard
 		if ! git checkout -q ${NIM_COMMIT} 2>/dev/null; then
+		  echo "Downloading Nim sources..."
```

Comment:
There is a mix of tabs and spaces throughout this file.

Reply:
Thanks for feedback. FIxed at https://github.com/status-im/nimbus-build-system/pull/71/commits/323c49461d29c59d702e6be1672049a7ea429362